docs: update the FJ extension section according to the UI changes (#24)
telerik/fiddler-jam-docs: extension/recording-a-log.md

1. Launch **Fiddler Jam** by clicking on the extension icon ![Extension main icon](../images/ext/ext-icons/small-logo.png).

> 1. (Optional) Expand the [**Advanced Options**]({%slug fj-capture-options%}) and choose the preferred settings. By default, all advanced options are turned on.

Reviewer: ```suggestion
1. (Optional) Expand [**Advanced Options**]({%slug fj-capture-options%}) and choose the preferred settings. By default, all advanced options are enabled.
```

`all advanced options are enabled` - will this continue to be the case in the future? Do we need this sentence?

> 1. Click **Start Capture** to begin recording. Fiddler Jam will start recording all network activity in your current browser tab. Click anywhere in the browser tab to hide the Fiddler Jam window and to continue recording. To bring the Fiddler Jam window back, click the extension's icon.

Reviewer: ```suggestion
1. Click **Start Capture** to start recording. Fiddler Jam will record all network activity in your current browser tab. Click anywhere in the browser tab to hide the Fiddler Jam window and continue recording. To show the Fiddler Jam window back, click the extension's icon.
```

   ![Start capture screen with the Advanced Options expanded](../images/ext/ext-images/extension-start-capturing-extended.png)

   >important Chrome displays an informational banner: **"Fiddler Jam" started debugging this browser**. Closing the notification or clicking **Cancel** won't stop the capturing session. Use the **Stop Capture** button (see the next step) to cancel an ongoing recording.
   ![Chrome warning banner](../images/ext/ext-images/extension-chrome-notification.png)

1. Click  **Stop Capture** to end the recording.

   ![Stop capturing screen](../images/ext/ext-images/extension-stop-capturing.png)

> 1. Before submitting the log, you could check the recorded log by clicking on **Capture Successful** line. The **Capture Details** screen will load with a list of the captured sessions, screenshots, console logs, etc. Go back to the previous screen to generate a link and submit the log to the cloud portal.

Reviewer: ```suggestion
1. Before submitting the recorded log, you can review it by clicking on the **Capture Successful** line. The **Capture Details** screen will load with a list of the captured sessions, screenshots, console logs, etc. Go back to the previous screen to generate a link and submit the log to the cloud portal.
```

   ![Stop capturing screen](../images/ext/ext-images/extension-link-options-show-sessions.png)

> 1. At this point, the captured traffic is recorded as a log but is not yet saved anywhere. Choose the appropriate [**Sharing Options**]({%slug fj-link-options%}), use the **Get Link** button and proceed with [submitting the recorded log to the JAM portal]({%slug fj-submitting-logs%}#submit-via-link).

Reviewer: `is not yet saved anywhere` sounds a bit confusing - is "saving" and "submitting / sharing" the same thing? What happens if I don't submit the log to the portal and close Fiddler Jam - is the log lost?

Author: Yep, your assumption is correct - not submitting the log will result in losing the recorded traffic.  I will provide extended description of the whole workflow and rethink using the whole **save** wording.

   ![Generate Link](../images/ext/ext-images/extension-link-options-password.png)

## Additional Recording Options

> During log recording, Fiddler Jam could capture not only the HTTP/HTTPS traffic but also:

Reviewer: `could` is overused and implies uncertainty. Prefer `can` as a simpler word.

```suggestion
During recording, Fiddler Jam monitors the HTTP/HTTPS traffic, but it can also capture:
```

- **Screenshots**: If the **_Take screenshots while capturing_** switch is enabled, a screenshot will be recorded every time you click on the loaded page.

- **Console logs**: If the **_Capture console_** switch is enabled, the log will contain all console logs executed from the source code.

>tip To reset the Fiddler Jam extension and start a new capturing session, use the **Capture Again** button.

> Additionally, Fiddler Jam provides switches for masking cookies, post data and for disabling the cache from [the Advanced options]({%slug fj-capture-options%}).

Reviewer: ```suggestion
Additionally, the Fiddler Jam [advanced options]({%slug fj-capture-options%}) allow you to mask cookies, post data and disable the browser cache.
```
